gard_owl_ingest/mondo_mapping_status.py

```diff
@@ -231,22 +233,55 @@ def gard_mondo_mapping_status(
             else 'skos:relatedMatch'
         chosen_mappings[term] = mappings_by_pred[pred][0]
 
+    # - sssom_df: Has less rows because of 'chosen mappings'. Just keeping 1 edge
     sssom_df = pd.DataFrame(chosen_mappings.values())
-    sssom_df = sssom_df.drop(columns=['proxy_id', 'mondo_predicate_id']).sort_values(by=['predicate_id', 'subject_id'])
-
-    outpath_mondo_sssom = RELEASE_DIR / f'gard-mondo{file_suffix}.sssom.tsv'
-    outpath_mondo_sssom_curate = RELEASE_DIR / f'gard-mondo{file_suffix}_curation.sssom.tsv'
-    write_tsv_with_comments(sssom_df, MONDO_SSSOM_METADATA_PATH, outpath_mondo_sssom)
 
     # - modify & save curation df
     sssom_curate_df = sssom_curate_df.drop('mapping_justification', axis=1)\
         .sort_values(by=['predicate_id', 'subject_id'])
+    outpath_mondo_sssom_curate = RELEASE_DIR / f'gard-mondo{file_suffix}_curation.sssom.tsv'
     write_tsv_with_comments(sssom_curate_df, MONDO_SSSOM_METADATA_PATH, outpath_mondo_sssom_curate)
 
+    # - gard-mondo.robot.template.sssom.tsv
```

Comment:
@matentzn I automated the creation of this `robot.template` for you, and I'm uploading to the [google sheet](https://docs.google.com/spreadsheets/d/1w5Xnzr5uNFcPrQqCT8mGBFHnGhwXDfQVzHKofw6kB7c/edit#gid=1648437389) now. However, now that I have discovered that there are in fact duplicate Mondo IDs in our exact match mappings, I would imagine we don't want to use this template to import anything in Mondo for now. If you already did that, I am sorry for giving you the wrong information on Thursday.

Reply:
@joeflack4 thanks for the thorough analysis. I made some [related tickets](https://github.com/monarch-initiative/mondo/issues?q=is%3Aissue+is%3Aopen+label%3AGARD) and will deal with this with Sabrina and Nicole. You can now move to the next issue on your list!

Reply:
Really appreciate, thanks!